Makefile

```diff
@@ -49,20 +49,20 @@ test-backend:
 	sh $(BACKEND_TEST_SCRIPT)
 	@echo "done"
 
+test-frontend:
+	python test/unit/run_tests.py -u -i
+
 # test-frontend-unit should use karma and jasmine to test
 # each of the Javascript components of the Narrative.
 # This is achieved through the grunt test invocation
 test-frontend-unit:
 	@echo "running frontend unit tests"
-	python test/unit/run_tests.py
+	python test/unit/run_tests.py -u
 	@echo "done"
 
-# test-frontend-e2e should use Selenium to perform an end-
-# to-end test of the front end components, with a running
-# Narrative system.
-test-frontend-e2e:
-	@echo "running frontend end-to-end tests"
-	cd $(FRONTEND_TEST_DIR)
+test-integration:
+	@echo "running integration tests"
+	python test/unit/run_tests.py -i
```

Comment:
To execute the integration tests locally, will we need to supply a token?

Reply:
Yes, I updated the docs to say so, and where to put it.